packages/sdk/src/animation/animationEaseCurves.ts

```diff
@@ -3,39 +3,44 @@
  * Licensed under the MIT License.
  */
 
+/** Control points of a cubic bezier curve. @see [[AnimationEaseCurves]]. */
+export type EaseCurve = [number, number, number, number];
+
 /**
  * Predefined ease curves from https://easings.net/. Curve shapes can be previewed there.
  * Create your own at http://cubic-bezier.com.
  * Note: Curves that end with "Back" include some overshoot.
  */
 export const AnimationEaseCurves = {
```

Comment:
Declaring this as
```ts
export const AnimationEaseCurves: { [index: string]: EaseCurve } = {
```
would make it strongly typed, and make the individual `as EaseCurve` casts unnecessary. In fact it would make it more strongly typed than it is now.

Reply:
I agree that from a coding perspective this is preferable. However, if something is declared as a lookup object like that, intellisense doesn't know about its contents, so doesn't autocomplete. Like this it's an inline type, and the keys are known from the type.

Reply:
I tested the syntax. Intellisense worked for me. 🤷‍♂️

Reply:
So how should we resolve this? I spent half my dev cycles without this prop autocompleting until I got around to debugging, so I'm not sure how it works for you.

Reply:
It was entirely untyped before. The signature I suggested adds type info. Can you try it locally and see if it works for you?

Reply:
I just tried this again. Added the type declaration `{ [name: string]: EaseCurve }`, recompiled, and Intellisense no longer autocompletes the properties of `MRE.AnimationEaseCurves` in functional tests. Note that even without this explicit type, it is not "untyped", i.e. `any`; it is implicitly typed to `{ Linear: EaseCurve; Step: EaseCurve; ... }`. I suspect that's the info autocomplete is using.

Reply:
If my suggestion is isn't working for you then its likely that it won't work for some of our user base either, so let's take your change.